static/app/components/sidebar/sidebarDropdown/index.tsx

```diff
@@ -130,18 +142,22 @@ const SidebarDropdown = ({api, org, orientation, collapsed, config, user}: Props
                     </UserSummary>
 
                     <div>
-                      <SidebarMenuItem to="/settings/account/">
-                        {t('User settings')}
-                      </SidebarMenuItem>
-                      <SidebarMenuItem to="/settings/account/api/">
-                        {t('API keys')}
-                      </SidebarMenuItem>
-                      <Hook
-                        name="sidebar:organization-dropdown-menu-bottom"
-                        organization={org}
-                      />
-                      {user.isSuperuser && (
-                        <SidebarMenuItem to="/manage/">{t('Admin')}</SidebarMenuItem>
+                      {!hideInternalLinks && (
```

Comment:
Why are these being hidden? These views are all for the individual user and not their interactions with a single organization.

It could be frustrating to not be able to disable email notifications because a membership expired.

Reply:
@markstory the problem is even though they aren't necessarily org-specific pages, we end up hitting endpoints that are org-specific and we still show the left sidebar specific to the last loaded organization. I'm not really sure what we can do other than hide these user views.

Reply:
What org endpoints are we hitting in account settings or API keys?

Reply:
@EvanPurkhiser The subscription and broadcast endpoints for one

Reply:
isn't broadcast specific to some other part of the sidebar? Same with broadcasts

Reply:
@EvanPurkhiser white listed those endpoints so disabled members can access their account settings page